examples/skysim/config.yml

```diff
@@ -24,6 +24,27 @@ TXIngestRedmagic:
 TXSourceTrueNumberDensity:
     nz: 301
     zmax: 3.0
+    name: TXSourceTrueNumberDensity
```

Comment:
this is throughout, and Joe said it's a bug, might be a different PR tho

Reply:
Yep - needs a fix in ceci!

Reply:
Made an issue here: https://github.com/LSSTDESC/ceci/issues/100